src/semgrep_agent/main.py

```diff
@@ -234,7 +239,7 @@ def protected_main(
         to_server = "" if publish_url == "https://semgrep.dev" else f" to {publish_url}"
         click.echo(
             get_aligned_command(
-                "manage", f"logged in{to_server} as deployment #{sapp.deployment_id}"
+                "manage", f"logged in{to_server} as {sapp.deployment_name}"
```

Comment:
```suggestion
                "manage", f"logged in{to_server} as org '{sapp.deployment_name}'"
```

Optional change, to communicate that you authenticate as an organization, not as a user. Might be more confusing this way though.

Reply:
Yeah haha I actually did this first and it looks really weird when it says "as org chmccreery" or "as org returntocorp". But I hear you... maybe if we went all the way and said "signed into the organization returntocorp" that would be best?